iphone/Classes/TiUIWebView.m

```diff
@@ -712,6 +722,20 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
 	}
 
 	NSURL * newUrl = [request URL];
+    
+    if (blacklistedURLs && blacklistedURLs.count > 0) {
+        NSString *urlAbsStr = [newUrl absoluteString];
+        
+        for (NSString *blackListedUrl in blacklistedURLs) {
+            if([urlAbsStr rangeOfString:blackListedUrl options:NSCaseInsensitiveSearch].location !=NSNotFound) {
+                if([self.proxy _hasListeners:@"onStopBlacklistedUrl"]) {
+                    NSDictionary *eventDict = [NSDictionary dictionaryWithObjectsAndKeys:urlAbsStr,@"url",@"Webview did not load blacklisted url.", @"messsage", nil];
```

Comment:
Try to use the shorthand dictionary next time. It looks more solid to read (in my personal opinion). Like this:
```objc
[[self proxy] fireEvent:@"onStopBlacklistedUrl" withObject:@{
    @"url": urlAbsStr,
    @"message": @"Webview did not load blacklisted url."
}];

And is the message the same as used on Android? Maybe we can unify that as well.